pennylane/templates/broadcast.py

```diff
@@ -572,5 +572,4 @@ def circuit(pars):
 
     # broadcast the unitary
     for wires, pars in zip(wire_sequence[pattern], parameters):
```

Comment:
What follows below are the deletions of the templates refactor. Templates can now pass their wires arguments through the operations they call.